### PR TITLE
WW-174 add logging after send event

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/WW-174

### CODE REVIEW

**Comment by cormorando** [comment]
LGTM

File: extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php [hunk 1]
```diff
@@ -25,6 +25,12 @@ public function appreciate() {
 			// send email that user received appreciation.
 			// Currently disabled because of: https://wikia-inc.atlassian.net/browse/WW-172
 			// $this->sendMail( $revisionId );
+
+			\Wikia\Logger\WikiaLogger::instance()->info('ContributionAppreciationMessage email send', [
+				'revision_id' => $revisionId,
+				'wiki_id' => $wgCityId,
+				'user_id' => $wgUser->getId()
```
Comment (rybmat) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: user_id stands for "from who" or "to who"? maybe it is worth to log both?

Comment (sebastianmarzjan) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: I think that wiki_id and user_id are logged by default, see:
Wikia\Logger\Hooks::onWikiFactoryExecute
and
Wikia\Tracer\WikiaTracer::updateContext

Comment (dianafa) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: `user_id` in default fields mentioned by Seba will mean "from who".

File: extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php [hunk 2]
```diff
@@ -25,6 +25,11 @@ public function appreciate() {
 			// send email that user received appreciation.
 			// Currently disabled because of: https://wikia-inc.atlassian.net/browse/WW-172
 			// $this->sendMail( $revisionId );
```
Comment (WikiaTech) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: MAJOR Remove this commented out code. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 19 issues
* MAJOR 15 major
* MINOR 4 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ContributionAppreciationController.class.php#L81](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L81): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L98](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L98): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L106](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L106): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L116](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L116): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L118](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L118): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L120](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L120): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L129](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L129): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L145](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L145): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)
1. MAJOR [ContributionAppreciationController.class.php#L150](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L150): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L166](https://github.com/Wikia/app/blob/ee329fed3d9e32908f443bc3a48f58b23cef4b0d/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L166): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)

**Comment by rybmat** [comment]
LGTM

**Pull request merged**